geschaeftsordnung-fachschaftsrat.md

```diff
@@ -105,11 +104,12 @@ Bei der Anwesenheit von Mitgliedern des Fachschaftsrates während seiner Sitzung
 
 (1) Das Protokoll jeder Sitzung muss mindestens die folgenden Punkte enthalten:
 
-1. Datum
-2. Anwesende Mitglieder
-3. Sitzungsleitung
-4. Protokollführung
-5. Beschreibung der Anträge und Beschlüsse mit genauen Abstimmungsergebnissen
+1. Tag, Zeit und Ort der Sitzung
```

Comment:
Dazu: Was wäre eigentlich der Ort einer Sitzung im Falle einer Onlinesitzung?

Reply:
Ich würde erwarten, dass in diesem Fall "Online" es tun sollte.